samples/commons/js/LinkedList.js

```diff
@@ -24,6 +28,8 @@ class LinkedList {
     constructor() {
         this.head = null;
         this.tail = null;
+        // initiate size of list to 0
+        this.sizeOfList = 0;
```

Comment:
Since this can be taken as sort of a private property, good convention to start with an underscore; and you could just call it `this._size`.

Reply:
Awesome, I will do that.